echo-web/config/echo.yml

```diff
@@ -12,8 +12,8 @@ front50:
 
 services:
   fiat:
-    enabled: ${services.fiat.enabled:false}
-    baseUrl: ${services.fiat.baseUrl:http://localhost:8089}
+    enabled: false
```

Comment:
Hrmm isn't this the config that allows overrides?  I cna't recall the impact of fixing this to false on this file...

Reply:
Actually think probably redundant b/c of the way this operates in newer spring... ignore this ;)

Reply:
Maybe it's only spring 2.6 that complains about the circular reference.  There are truckloads of precedence rules, I expect existing overrides to keep working.